TAInformation/urls.py

```diff
@@ -1,8 +1,9 @@
 from django.urls import path
-
+from django.contrib import admin
 from . import views
 
 urlpatterns = [
```

Comment:
It looks like you don't have the updated urls as well. Using the Pull feature in PyCharm and resolving merge conflicts should clear this up though

Reply:
I don't know about this, I updated my branch from the recent change  in master